script/github-actions/get-changed-apps.js

```diff
@@ -95,6 +97,26 @@ const getChangedAppsString = (
   return [...new Set(appStrings)].join(delimiter);
 };
 
+/**
+ * Checks whether all file paths belong to apps on the allowlist with continuous deployment
+ * enabled. Returns false if all apps don't have continuous deployed enabled.
+ *
+ * @param {string[]} filePaths - A list of relative file paths.
+ * @param {Object} config - Changed apps config.
+ * @returns {Boolean} Whether apps of file paths have enabled continuous deployment.
+ */
+const isContinuousDeploymentEnabled = (filePaths, config) => {
```

Comment:
Originally, I was going to add this functionality to the `getChangedAppsString` function as another `if` statement when the `outputType` is `continuous-deployment`, but I felt like it didn't fit with the purpose of the function. `getChangedAppsString` returns a string of app information, whereas this function is determining if all allowed apps have `continuousDeployment` set to `true`.